docs/user-guide/security-features.md

```diff
@@ -1,156 +1,228 @@
+{% hint style="info" %}
+Vulnerability information will only be displayed for images that have undergone vulnerability scanning. If no vulnerabilities were identified during the scan, the **Security** tab will display a zero count, indicating **Security (0)**.
+{% endhint %}
+
 
 ### Accessing Vulnerability Information on the App Details Page
 
-Devtron offers the capability to identify vulnerabilities even after an image has been deployed. By navigating to the `app details` page, you can find comprehensive details about the vulnerabilities associated with the deployed image.
+Devtron offers the capability to identify vulnerabilities even after an image has been deployed. By navigating to the `App Details` page, you can find comprehensive details about the vulnerabilities associated with the deployed image.
 
 With this capability, Devtron empowers users to stay informed about the security vulnerabilities present in their deployed images.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/vulnerability-app-details.jpg)
+![Figure 6: App Details Tab](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/vulnerability-app-details.jpg)
+
+Clicking the 'Details' link in the security vulnerabilities report (shown above) reveals detailed information about those found within the deployed image.
 
-By clicking on the 'Details' link in the security vulnerabilities report, you can access detailed information about the security vulnerabilities present inside the deployed image.
+![Figure 7: Detailed List of Vulnerabilities](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/detailed+vulnerability-app-details.jpg)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/detailed+vulnerability-app-details.jpg)
+---
 
 ## Configuring Security Policies
 
 You can establish security policies for their vulnerabilities through the `Security Policies` tab, which can be accessed from the left pane by navigating to `Security` and selecting `Security Policies`. Policies are implemented in a hierarchical order, following a specific sequence. The order of implementation is as follows, starting from the highest level:
 
-* **Global**
-* **Cluster**
-* **Environment**
-* **Application**
+* Global
+* Cluster
+* Environment
+* Application
 
-Policies are implemented in a hierarchical order, with the following sequence: Global, Cluster, Environment, and Application. Higher-level policies take precedence over lower-level policies, ensuring a systematic and structured enforcement of security measures.
+Higher-level policies take precedence over lower-level policies, ensuring a systematic and structured enforcement of security measures.
```

Comment:
This statement seems unclear/incorrect.
Referring the list, the high to low order looks like (Global > Cluster > Environment > Application).
Considering the statement the takeaway would be Policy defined at Global level would have higher priority than policy defined at Application + Environment level, which is not true.

Rather, policy defined at Application + Environment level would have higher precedence over what is defined at Global / Cluster / Environment level.

P.S. The policy at application level is defined for each environment in that application.

Reply:
Corrected